datafusion/physical-expr/src/window/row_number.rs

```diff
@@ -61,6 +58,24 @@ impl BuiltInWindowFunctionExpr for RowNumber {
         &self.name
     }
 
+    fn add_equal_orderings(&self, builder: &mut OrderingEquivalenceBuilder) {
+        // Only the built-in `RowNumber` window function introduces a new
```

Comment:
Maybe we can replace comment 
`// Only the built-in RowNumber window function introduces a new` with 
`// The built-in RowNumber window function introduces a new`. Since method implements it (and special handling is not done outside), we do not need stress this is the only window function with this property.

Reply:
👍 done in ad165028d5